src/features/operator/Fee/FeeWithdraw/index.tsx

```diff
@@ -76,6 +111,9 @@ export const FeeWithdraw = (): JSX.Element => {
             wrapperCol={{ span: 24 }}
             validateTrigger="onBlur"
             onFinish={onFinish}
+            onValuesChange={(_, values) => {
```

Comment:
Could we move `setFeeWithdrawAmount()` in InputAmount `setInputValue`instead?

Reply:
The benefit of onValuedChanged is that the price is getting updated as the user is typing.

setInputValue only really executes under certain circumstances.

Reply:
Right